lib/go/edgecontext/req_context.go

```diff
@@ -91,6 +91,16 @@ func (e *EdgeRequestContext) CountryCode() string {
 	return e.raw.CountryCode
 }
 
+// LanguageCode returns the ISO 639-1 language identifier for the client
+func (e *EdgeRequestContext) LanguageCode() string {
+	return e.raw.LanguageCode
+}
+
+// RegionCode returns the ISO 3166-1 region identifier for the client
+func (e *EdgeRequestContext) RegionCode() string {
+	return e.raw.RegionCode
+}
```

Comment:
I don't think you added them to https://github.com/reddit/edgecontext/blob/e199d71df66ebee2f58720d9d3ed1c507953388e/lib/go/edgecontext/edgecontext.go#L118

Reply:
Thanks, updated.